slp2mp4/gui_test.py

```diff
@@ -0,0 +1,289 @@
+import tkinter as tk
+from tkinter import filedialog
+import subprocess
+import os
+from config import Config
+import json
+
+#some notes: slippi folder is the folder containing slippi replays
+
+
+
+def browse_melee_iso():
+    melee_iso_path = filedialog.askopenfilename(title="Select Melee ISO file", filetypes=[("ISO files", "*.iso")])
```

Comment:
For many of these functions, consider adding either `initialdir` or `initialfile` with data from `config.json` to make things faster